vue/components/forms/personalization-form/reference.vue

```diff
@@ -194,6 +195,19 @@ export const Reference = {
         if (this.onPostConfig) this.$bus.unbind(this.onPostConfig);
     },
     methods: {
+        hasValidInput(initials) {
+            if (!initials && initials !== 0) return true;

```

Comment:
we need to remove this validation from here and move this into a centralized validation execution code

Reply:
@joamag moved to the disabling of the button as requested:

![](http://g.recordit.co/pAeFaR5VlP.gif)